examples/video/instream/videoModule/videojs/video-module-videojs.html

```diff
@@ -0,0 +1,280 @@
+                            },
+                        },]
+                    },
+                    cache: {
+                        url: 'https://prebid.adnxs.com/pbc/v1/cache'
+                    },
+                    targetingControls: {
+                        allowTargetingKeys: ['BIDDER', 'AD_ID', 'PRICE_BUCKET', 'SIZE', 'DEAL', 'SOURCE', 'FORMAT', 'UUID', 'CACHE_ID', 'CACHE_HOST', 'ADOMAIN']
+                    },
+                });
+
+                pbjs.addAdUnits(adUnits);
+
+                pbjs.onEvent('videoSetupComplete', e => {
+                    // Load media with its Metadata when the video player is done instantiating.
+                    videojs('player').loadMedia({
+                        id: 'XYXYXYXY',
+                        src: 'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/SubaruOutbackOnStreetAndDirt.mp4',
+                        title: 'Subaru Outback On Street And Dirt',
+                        description: 'Smoking Tire takes the all-new Subaru Outback to the highest point we can find in hopes our customer-appreciation Balloon Launch will get some free T-shirts into the hands of our viewers.',
+                        type: 'video/mp4'
+                    });
+                });
+
+                pbjs.onEvent('videoSetupFailed', e => {
+                    console.log('The player setup failed: ', e);
+                });
+
+                pbjs.onEvent('videoAdRequest', (e) => {
+                    console.log('An ad request was made: ', e);
+                });
+
+                pbjs.onEvent('videoBidError', e => {
+                    console.log('An Ad Error came from a Bid: ', e);
+                });
+
+                pbjs.onEvent('videoBidImpression', e => {
+                    console.log('An Ad Impression came from a Bid: ', e);
+                });
+
+                pbjs.requestBids(adUnits);
+            });
+        &lt;/script&gt;
+
+                </pre>
+              </div>
+
+              <h4>Place this code in the page body.</h4>
+              <div class="pb-code-hl-wrap">
+                <!--body code example-->
+                <pre class="pb-code-hl" >
+                  <!--put body html and javascript here-->
+                  &lt;div class="example-video-container"&gt;
+                    &lt;video id="player" class="vjs-big-play-centered"&gt;&lt;/video&gt;
+                  &lt;/div&gt;
+                </pre>
+              </div>
+            `;
+        </script>
+    </div>
+</div>
+
+{% if jekyll.environment == "production" %}
+<script>
+    window.onetrustLoaded.then(() => {
+        OneTrust.InsertHtml(exampleHTML, 'videoExample', null, {deleteSelectorContent: true}, 'C0003');
+    });
+</script>
+{% else %}
+<script>
+    const videoExampleDiv = document.getElementById('videoExample');
+    videoExampleDiv.innerHTML = exampleHTML;
```

Comment:
Here the code that actually executes and the code that is displayed as the example are different. Are you able to update this so that the code in the example HTML and the code that actually executes are one and the same? Here's an example:
https://github.com/prebid/prebid.github.io/blob/master/examples/video/outstream/basic-ima.html#L155